copilot.sh

```diff
@@ -5,8 +5,13 @@ set -o errexit
 set -o nounset
 
 # This script requires https://github.com/charmbracelet/gum
-command -v /usr/bin/gum >/dev/null 2>&1 \
-  || { echo >&2 "This script requires https://github.com/charmbracelet/gum, but it's not installed.  Aborting."; exit 1; }
```

Comment:
`command -v gum >/dev/null 2>&1` should work (no path). I just tested that on my mac.

Reply:
Agreed, this is probably just the full path causing the problem.

Reply:
I updated! Thanks for reviewing!